prow/knative/plugins.yaml

```diff
@@ -94,6 +94,16 @@ config_updater:
 blunderbuss:
   use_status_availability: true
 
+# Settings for the "trigger" plugin.
+triggers:
+- repos:
```

Comment:
It looks `repos` should be a list of strings - https://github.com/kubernetes/test-infra/blob/e47f01cbed7a926b20f1dd4b7e18d06a6d9bc146/prow/plugins/config.go#L429